types/url-file-size/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for url-file-size 1.0
+// Project: https://github.com/dimdenGD/url-file-size#readme
+// Definitions by: Shehab Ali Tweel <https://github.com/shahoob>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference lib="DOM" />
```

Comment:
This doesn't seem right; the readme of the package shows using Node's URL, not the browser URL.

Reply:
yes, i don't know how do i tell DefinitelyTyped that i need @types/node.

Reply:
```ts
/// <reference types="node" />

import { Url } from 'url';
```

Reply:
ok thanks

Reply:
You imported it, but you aren't actually using it; not sure how that compiles. maybe you don't need either?

Reply:
the source code uses the URL class, not the interface.
i updated the definition files to reflect that.